roles/candlepin/templates/candlepin.conf.j2

```diff
@@ -20,10 +20,10 @@ candlepin.async.jobs.ExpiredPoolsCleanupJob.schedule=0 0 0 * * ?
 
 jpa.config.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 jpa.config.hibernate.hbm2ddl.auto=validate
-jpa.config.hibernate.connection.username=candlepin
+jpa.config.hibernate.connection.username={{ candlepin_db_user }}
 jpa.config.hibernate.connection.password={{ candlepin_db_password }}
 jpa.config.hibernate.connection.driver_class=org.postgresql.Driver
-jpa.config.hibernate.connection.url=jdbc:postgresql://{{ candlepin_db_host }}:{{ candlepin_db_port }}/candlepin{% if candlepin_db_ssl %}?ssl=true{% endif %}{% if candlepin_db_ssl and candlepin_db_ssl_ca is defined %}&sslrootcert={{ candlepin_db_ssl_ca }}{% endif %}{% if not candlepin_db_ssl_verify and candlepin_db_ssl %}&sslfactory=org.postgresql.ssl.NonValidatingFactory{% endif %}
+jpa.config.hibernate.connection.url=jdbc:postgresql://{{ candlepin_db_host }}:{{ candlepin_db_port }}/{{ candlepin_db_name }}{% if candlepin_db_ssl_mode %}?sslmode={{ candlepin_db_ssl_mode }}{% endif %}{% if candlepin_db_ssl_ca is defined %}&sslrootcert={{ candlepin_db_ssl_ca }}{% endif %}
```

Comment:
We also have a default and can simplify this. That also resolves the issue that if `candlepin_db_ssl_ca` is defined but `candlepin_db_ssl_mode` isn't the URL is invalid. And I wonder if `candlepin_db_ssl_ca` isn't always defined since it's mentioned in the defaults. Perhaps like this?
```suggestion
jpa.config.hibernate.connection.url=jdbc:postgresql://{{ candlepin_db_host }}:{{ candlepin_db_port }}/{{ candlepin_db_name }}?sslmode={{ candlepin_db_ssl_mode }}{% if candlepin_db_ssl_ca %}&sslrootcert={{ candlepin_db_ssl_ca }}{% endif %}
```

The same can be applied below for `org.quartz.dataSource.myDS.URL`.